app1/src/components/fields/InputField.js

```diff
@@ -43,18 +43,22 @@ function getStyles(name, personName, theme) {
 
 export default function MultipleSelect({displayedName,setDisplayedName,getNewPersonName}) {
   const theme = useTheme();
+  const [labelDisplayed, setLabelDisplayed] = React.useState('Name');
```

Comment:
ttry to use the 'useState' as I used it on App.js and check that not error are displayed. I changed it on purpose because on most proyects is used in that way.